src/styles/main.scss

```diff
@@ -102,7 +106,7 @@ h1 {
     a {
         color: $blink;
         text-decoration: none;
-        font-family: 'Droid Sans', sans-serif;
+        font-family: minecraftia;
```

Comment:
Actually, minecraftia was a rushed workaround for last edition – it was never meant to stay.